arches_querysets/querysets.py

```diff
@@ -68,7 +68,7 @@ def with_node_values(
             Fine-quality calf or lamb parchment ...
 
         as_representation:
```

Comment:
as_representation seems a little vague.  maybe `as_json` is better?

Reply:
The DRF docs tend to use "representation" to be content-type agnostic (XML, or HTML via HTMX), so `as_native_python_type()` would be a little more precise but then cause confusion with my use of `to_python()` elsewhere. I'm not quite sure what to do here. I like that "representation" is clear if you're coming from a DRF background.